repo/packages/H/hdfs/0/command.json

```diff
@@ -0,0 +1,5 @@
+{
+  "pip": [
+    "git+https://github.com/mesosphere/dcos-hdfs.git#dcos-hdfs=0.1.0"
```

Comment:
We may not change the versions on a weekly basis or release that frequently. We recently released 0.1.0 as a public beta release. Because of that, our system may need to be different than for Cassandra.